python/vyos/utils/process.py

```diff
@@ -139,13 +164,18 @@ def cmd(command, flag='', shell=None, input=None, timeout=None, env=None,
     expect:  a list of error codes to consider as normal
     """
     decoded, code = popen(
-        f'{auth} {command}'.strip(), flag,
+        command, flag,
         stdout=stdout, stderr=stderr,
         input=input, timeout=timeout,
         env=env, shell=shell,
         decode=decode,
+        auth=auth,
+        vrf=vrf,
+        netns=netns,
     )
     if code not in expect:
+        wrapper = get_wrapper(vrf, netns, auth='')
```

Comment:
```suggestion
        wrapper = get_wrapper(vrf, netns, auth)
```

You overwrite `auth` in this statememnt. If one passes in `auth` to `cmd()` you will reset it to `''` in this line.

Reply:
In this line we generate command without auth just to show it on the CLI on error